docs/source/getting_started/10_minutes_to_graviti.rst

```diff
@@ -4,3 +4,182 @@
 #######################
  10 Minutes to Graviti
 #######################
+
+This is a simple introductory tutorial for beginners.
+
+**************
+ Registration
+**************
+
+Before using Graviti SDK, please finish the following registration steps:
+
+- Please visit `Graviti AI Service(GAS)`_ to sign up.
+- Please visit `Graviti Developer Tools`_ to get an AccessKey.
+
+.. _graviti ai service(gas): https://gas.graviti.com/tensorbay/
+.. _Graviti Developer Tools: https://gas.graviti.com/tensorbay/developer
+
+.. note::
+   An AccessKey is needed to authenticate identity when using Graviti via SDK.
+
+*********************
+ Dataset Preparation
+*********************
+
+This step is only for users who do not have datasets in their workspace. By running the code
+below, users can create a very simple dataset to experience Graviti SDK.
+
+.. code:: python
+
+   from graviti import DataFrame, Workspace
+   import graviti.portex as pt
+
+   ws = Workspace(f"{YOUR_ACCESSKEY}")
+   dataset = ws.datasets.create("Graviti-dataset-demo")
+   draft = dataset.drafts.create("Draft-1")
+
+   ob = pt.build_openbytes("main")
+   box2ds = ob.label.Box2D(
+       categories=["boat", "car"],
+       attributes={"difficult": pt.boolean()},
+   )
+   voc2012 = pt.record(
+       {
+           "filename": pt.string(),
+           "box2ds": pt.array(box2ds),
+       }
+   )
+
+   filenames = ["a.jpg", "b.jpg", "c.jpg"]
+   data = []
+   for filename in filenames:
+       row_data = {
+           "filename": filename,
+           "box2ds": DataFrame(
+               [
+                   {
+                       "xmin": 1,
+                       "ymin": 1,
+                       "xmax": 4,
+                       "ymax": 5,
+                       "category": "bird",
```

Comment:
category `bird` mismatched with the schame `categories=["boat", "car"]`, and `attribute` also.